### PR TITLE
[v1.89][OSSM-6967] Clean up created resources in istio namespace after test + Fix for cypress flakiness on OCP 4.16

### PR DESCRIPTION
**Pull request by mkralik3**
### Describe the change

Backports of 

- https://github.com/kiali/kiali/pull/7642
- https://github.com/kiali/kiali/pull/7625

### CODE REVIEW

File: frontend/cypress/integration/featureFiles/overview.feature [hunk 1]
```diff
@@ -81,8 +81,10 @@ Feature: Kiali Overview page
     And the "healthy" application indicator should list the application
 
   @sleep-app
+  @error-rates-app
```
Comment (ScriptingShrimp) [inline frontend/cypress/integration/featureFiles/overview.feature:84]: @mkralik3 checking against https://github.com/kiali/kiali/pull/7625/files there is bookinfo-app and not error-rates-app

**Review by ScriptingShrimp — COMMENTED** [review]
LGTM

File: frontend/cypress/integration/featureFiles/overview.feature (hunk 1 above)
Comment (ScriptingShrimp) [reply frontend/cypress/integration/featureFiles/overview.feature:84]: oh I see error-rates-app in master also. I guess than its fine!

Comment (mkralik3) [reply frontend/cypress/integration/featureFiles/overview.feature]: fixed

**Review by ScriptingShrimp — APPROVED** [review]
LGTM!

**Pull request merged**